pkg-py/src/shinychat/_chat_types.py

```diff
@@ -26,6 +26,12 @@ def __init__(
     ):
         self.role: Role = role
 
+        is_html = (
+            isinstance(content, HTML)
+            or isinstance(content, TagList)
+            or isinstance(content, Tagifiable)
+        )
```

Comment:
Most importantly, this adds `Tag`

```suggestion
        is_html = isinstance(content, (Tag, TagList, HTML, Tagifiable))
```

Reply:
oh right, thanks!